src/styles/card.scss

```
@@ -102,6 +102,8 @@
     text-decoration: none;
     text-transform: uppercase;
 
+    transition: 1s;
```

Те ж саме, вказуй конкретно, чому хочеш зробити перехід
```suggestion
    transition: background-color 1s;
```